source/guide_conduit.rst

```diff
@@ -0,0 +1,215 @@
+ # and room ids. Examples: matrix.org, conduit.rs
+ # The Conduit server needs to be reachable at https://your.server.name/ on port
+ # 443 (client-server) and 8448 (federation) OR you can create /.well-known
+ # files to redirect requests. See
+ # https://matrix.org/docs/spec/client_server/latest#get-well-known-matrix-client
+ # and https://matrix.org/docs/spec/server_server/r0.1.4#get-well-known-matrix-server
+ # for more information
+ 
+ #server_name = "<username>.uber.space"
+ 
+ # This is the only directory where Conduit will save its data
+ database_path = "/home/<username>/conduit_data/conduit.db"
+ 
+ port = 6167
+ 
+ # Max size for uploads
+ max_request_size = 20_000_000 # in bytes
+ 
+ # Disable registration. No new users will be able to register on this server
+ allow_registration = true
+ 
+ # Disable encryption, so no new encrypted rooms can be created
+ # Note: existing rooms will continue to work
+ #allow_encryption = false
+ allow_federation = true
+ 
+ # Enable jaeger to support monitoring and troubleshooting through jaeger
+ #allow_jaeger = false
+ 
+ trusted_servers = ["matrix.org"]
+ 
+ #max_concurrent_requests = 100 # How many requests Conduit sends to other servers at the same time
+ #log = "info,state_res=warn,rocket=off,_=off,sled=off"
+ #workers = 4 # default: cpu core count * 2
+ 
+ address = "0.0.0.0" # This makes sure Conduit can only be reached using the reverse proxy
+ 
+ proxy = "none"
+ 
+ # The total amount of memory that the database will use.
+ db_cache_capacity_mb = 100
+
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/conduit.ini`` with the following content:
+
+.. warning:: Replace all place holders such as ``<username>`` with your actual values!
+
+.. code-block:: ini
+ :emphasize-lines: 2,3
+
+ [program:conduit]
+ command=/home/<username>/conduit
+ environment=CONDUIT_CONFIG="/home/<username>/conduit.toml"
+ process_name=%(program_name)s
+ autostart=true
+ startsecs=5
+ startretries=3
+ stopwaitsecs=5
+
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Configure web server
+--------------------
+
+.. note:: 
+  To talk to each other on Matrix, your chat client often needs to know how to talk to users on different servers.
+  When a Matrix chat client encounters the user ``@isabell:uberspace.de`` it will look at ``https://uberspace.de/.well-known/matrix/server``,
+  which contains information about where to find the actual Matrix server for that domain (``uberspace.de``).
+  
+  This is mostly practical for bigger Matrix servers with many users, where you don't want to run the website (uberspace.de) on the same hardware as the matrix server.
+  In this guide however, we will just host the Matrix server and your potential website side by side on the same uberspace account.
+
+Add .well-known files to point to the same domain:
+
+::
+
+  [isabell@stardust ~]$ mkdir -p ~/html/.well-known/matrix/
```

Comment:
Please don't echo stuff into files, just tell users to edit the file like you did with the other ones above as stated in our [style guide](https://github.com/Uberspace/lab/blob/main/STYLE.md).

Reply:
Done.